packages/grpc-js/src/compression-algorithms.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+export const CompressionAlgorithms = {
```

Comment:
What is the benefit of using `const` here instead of `enum`? I think an enum would be better for helping the user populate the channel option, because they could use `CompressionAlgorithm.gzip`, for example.

Reply:
Generally I disprefer enums in Typescript because they're one of the few language features which aren't runtime-only, and they interact somewhat poorly with the rest of the type system, since they're somewhat poorly supported and the language designers seem to pushing people in the direction of [using object literals (with `as const`) instead](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums).

Those probably aren't huge concerns in this case; I don't see a way that making it an enum would frustrate a typical user of the library.  For user convenience and consistency with the way other such options are handled, I'll change it to an enum.